osmaxx/api_client/conversion_api_client.py

```diff
@@ -114,7 +124,7 @@ def _create_job_TODO_replace_me(self, extraction_order, request):  # noqa
             }
         })
         try:
-            response = self.authorized_post(CONVERSION_JOB_URL, json_data=request_data)
+            response = self.authorized_post(OLD_CONVERSION_JOB_URL, json_data=request_data)
```

Comment:
Yes, `_create_job_TODO_replace_me` should eventually be removed, once we've switched to `extraction_order.forward_to_conversion_service` in the view.